library/Fission/IPFS/Info/Types.hs

```diff
@@ -1,4 +1,11 @@
-module Fission.IPFS.Info.Types (Info (..)) where
+module Fission.IPFS.Info.Types
+  ( Info (..)
+  , id'
+  , publicKey
+  , addresses
+  , agentVersion
+  , protocolVersion
```

Comment:
It was complaining about us not exporting the lenses. `id` conflicts with `RIO.id :: a -> a` which is a super fundamental function that really anyone would assume to be available... so switching to `id'`